policyengine_us/parameters/gov/states/oh/tax/income/rates/agi.yaml

```diff
@@ -0,0 +1,43 @@
+description: OH individual tax on nonbusiness income
+brackets:
+  - threshold:
+      0000-01-01: 0
+    base_amount: 
+      0
+    rate:
+      2021-01-01: 0
+  - threshold: 
+      0000-01-01: 25_001
+    base_amount:
+      2021-01-01: 346.16
```

Comment:
We don't need `base_amount` since it's computed mechanically from the rate structure. You can remove all those values.

Reply:
After double checking this, I believe the base amount applies independently of the rate structure.

If you go to page 32 of the tax form ([link to the specific page here](https://tax.ohio.gov/static/forms/ohio_individual/individual/2021/pit-it1040-booklet.pdf#page=32)) it gives a table, and you can see that at $25001 it starts with $347 as a base amount, and then increases in line with the rate on top of that. The table directly above that on page 31 seems clear that it's base amount + rate as well. At each new threshold, the stated base kicks in + the rate above as well. 

Let me know if I'm missing something or if there's a better way to represent this.

Reply:
Clarification: the brackets above the second one appear to be computable from the rate structure. But the initial $346.16 is not.